test/exe/require_static.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copied from https://github.com/envoyproxy/envoy/blob/a12869fa9e9add4301a700978d5489e6a0cc0526/test/exe/envoy_static_test.sh.
```

Comment:
A few words in the beginning to explain the purpose of the script? something like `this scripts checks if the provided binary is static linked. Fails if not.`.

Reply:
never mind. i think this one is pretty self explained.